src/api/l_graphics_shader.c

```diff
@@ -28,11 +28,12 @@ static int l_lovrShaderClone(lua_State* L) {
       case LUA_TNUMBER: flag.id = lua_tointeger(L, -2); break;
       default: lovrThrow("Unexpected ShaderFlag key type (%s)", lua_typename(L, lua_type(L, -2)));
     }
+    lovrCheck(flags.length < ~((uint32_t)0), "Too many flags");
```

Comment:
I'd prefer writing this constant as `~0u` to match the rest of the codebase (though this particular one might be better as a smaller constant limit as mentioned above).